### PR TITLE
#159388713 Delivers user account verification

### PR DESCRIPTION
**Pull request by chukwuemekachm**
#### What does this PR do?
- Completes feature to verify user account on successful registration using email
#### Description of Task to be completed?
- Write unit tests for emailSending
- Integrate email sending using sendgrid
- Write tests for JwtHelper
- Implement JwtHelper
- Write tests for verify account
- Add verifyAccount method to verify user account
- Write tests for resend verification link
- Add resendVerificationLink method to resend verification link
- Test endpoints using POSTMAN
- Update `.env-sample` file
- Update `.travis.yml` file
#### How should this be manually tested?
- Clone the repository
- Start up the command line /Terminal
- Change to the repo directory
- Run `npm install`
- Run `npm run migrate`
- Run `npm run start:dev`
- Use POSTMAN to create a GET request to `/api/auth/verify?token=n8c832x1i2932`
- Use POSTMAN to create a POST request to `/api/auth/verify`
- POST request payload
```
{
  email: user@test.com
}
```
#### Any background context you want to provide?
- Create a Postrgresql database before you start
- Create a `.env` file and fill it with the necessary variables as specified in `.env-sample`
#### What are the relevant pivotal tracker stories?
[159388713](https://www.pivotaltracker.com/story/show/159388713)
#### Screenshots (if appropriate)

screen shot 2018-08-16 at 12 39 53 pm

#### Questions:
- None

### CODE REVIEW

File: .env-sample [hunk 1]
```diff
@@ -6,3 +6,7 @@ DB_DATABASE=<string>
 DB_DIALECT=<string>
 
 DATABASE_URL=<string>
+
+SENDGRID_API_KEY=<string>
```
Comment (paradisekelechi) [inline .env-sample:10]: Your use of env variables is good

File: package.json [hunk 2]
```diff
@@ -4,36 +4,42 @@
...
   },
   "repository": {
     "type": "git",
     "url": "git+https://github.com/chukwuemekachm/Shopmon.git"
   },
   "keywords": [],
-  "author": "",
-  "license": "ISC",
+  "author": "Chima Chukwuemeka",
+  "license": "MIT",
```
Comment (paradisekelechi) [inline package.json:22]: Why the change to MIT?

File: server/services/mailTemplates.js [hunk 3]
```diff
@@ -0,0 +1,13 @@
+const mailTemplates = {
```
Comment (paradisekelechi) [inline server/services/mailTemplates.js:1]: I'm impressed with this!

File: package.json (hunk 2 above)
Comment (chukwuemekachm) [reply package.json:22]: The badge and license file on the repo specify MIT.

File: .env-sample (hunk 1 above)
Comment (chukwuemekachm) [reply .env-sample:10]: Thanks.

File: server/services/mailTemplates.js (hunk 3 above)
Comment (chukwuemekachm) [reply server/services/mailTemplates.js:1]: Thanks.
